pom.xml

```diff
@@ -27,8 +27,6 @@
     <module>packages/java/hilla-dev</module>
     <module>packages/java/hilla-dev-mode</module>
     <module>packages/java/hilla-react</module>
-    <module>packages/java/hilla-spring-boot-starter</module>
-    <module>packages/java/hilla-react-spring-boot-starter</module>
```

Comment:
Just asking: 

Now it would be breaking change in a minor version (well Hilla 2 -> 24 is major... so dismiss if not applicable) 

Otherwise you could let those modules still be alive and add maven relocation to say: hey it's deprecated; use the other artefacts: https://maven.apache.org/ref/3.2.1/maven-model/maven.html#class_relocation